src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx

```diff
@@ -47,9 +52,121 @@ describe('ServiceList', () => {
       expect(screen.getByText(service.name)).toBeInTheDocument()
     })
   })
+
+  test('it should send patch request correctly', async () => {
+    const patchMock = jest.fn()
+
+    useGetServicesMock.mockImplementation((): any => {
+      return {
+        data: mockService,
+        error: null,
+        loading: false,
+        refetch: jest.fn()
+      }
+    })
```

Comment:
Nit: I think you can write this a bit cleaner (same for everywhere else you've used `mockImplementation`

```suggestion
        useGetServicesMock.mockReturnValue({
            data: mockService,
            error: null,
            loading: false,
            refetch: jest.fn()
        })
```